download_file_request.go

```diff
@@ -55,6 +55,8 @@ func (request FileDownloadRequest) GetQuery() url.Values {
 
 	if request.IncludeOriginal {
 		query.Set("includeOriginalStrings", "true")
+	}else{
+		query.Set("includeOriginalStrings", "false")
```

Comment:
I'm not sure that we should explicit tell `false`. My expectation - if the parameter is not provided then its value `false`.
I want to talk with someone in File API team to confirm my assumption.

Reply:
I did some test with curl (with includeOriginalStrings not set) and it seems to me (maybe it is somewhere hidden in my project settings that I missed) that the default value is true here. but getting clarification from the team is definitely a good idea, maybe they can update the official doc for that too, thanks!